examples/launch_mb_mo_optimization.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+from pathlib import Path
+from typing import Dict, Any, Optional
+
+import numpy as np
+
+from syne_tune import Tuner, StoppingCriterion
+from syne_tune.backend import LocalBackend
+from syne_tune.config_space import randint, uniform
+from syne_tune.optimizer.baselines import BayesianOptimization
+from syne_tune.optimizer.schedulers import FIFOScheduler
+from syne_tune.optimizer.schedulers.multiobjective.multi_surrogate_multi_objective_searcher import (
+    MultiObjectiveMultiSurrogateSearcher,
+)
+from syne_tune.optimizer.schedulers.multiobjective.random_scalarization import (
+    MultiObjectiveLCBRandomLinearScalarization,
+)
+from syne_tune.optimizer.schedulers.searchers.bayesopt.models.estimator import Estimator
+
+
+def create_gaussian_process_estimator(
+    config_space: Dict[str, Any],
+    metric: str,
+    mode: Optional[str] = None,
+    random_seed: Optional[int] = None,
+    search_options: Optional[Dict[str, Any]] = None,
+) -> Estimator:
+    scheduler = BayesianOptimization(
+        config_space=config_space,
+        metric=metric,
+        mode=mode,
+        random_seed=random_seed,
+        search_options=search_options,
+    )
+    searcher = scheduler.searcher  # GPFIFOSearcher
+    state_transformer = searcher.state_transformer  # ModelStateTransformer
+    estimator = state_transformer.estimator  # GaussProcEmpiricalBayesEstimator
+
+    # update the estimator properties
+    estimator.active_metric = metric
```

Comment:
yes unfortunately we need this metric inside the estimator to gather the right outputs from results. This is not optimal but I could not find a better way to pass the metric names since they get automatically adjusted to `INTERNAL_METRIC_NAME`.

Reply:
I am also not very happy about this internal renaming (this is old code), but I so far did not find an easy way to refactor this, because this internal code should work also if no name is given (except multi-objective).

I'd leave it as is, as long as it works.